### PR TITLE
[DOCS] Diagnostic Pprof

### PR DESCRIPTION
**Pull request by lucabelluccini**
Clarify PProf settings must be enabled prior to collecting the diag with pprof.

I would kindly ask the documentation team to:
- Improve the wording and format
- Backport this to any applicable version

### CODE REVIEW

File: docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc [hunk 1]
```diff
@@ -315,7 +315,11 @@ Note that the diagnostics bundle is intended for debugging purposes only, its st
 IMPORTANT: The configuration files in the archive may have credentials in *plain text*.
 These will need to be manually redacted before the archive is shared.
 
-The diagnostics command is also able to collect `pprof` data in the archive if the `--pprof` flag is passed:
+The diagnostics command is also able to collect `pprof` data in the archive if the `--pprof` flag is added when collecting the diagnostic.
+
+To collect the `pprof` data:
+- Enable/add `agent.monitoring.pprof.enabled: false` to `elastic-agent.yml`
```
Comment (belimawr) [inline docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc:321]: Are you sure this should be false? I believe it should be true
```suggestion
- Enable/add `agent.monitoring.pprof.enabled: true` to `elastic-agent.yml`
```

Comment (lucabelluccini) [reply docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc:321]: 🤦 Thank you

Comment (lucabelluccini) [reply docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc:321]: Too much copy paste

File: docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc [hunk 2]
```diff
@@ -315,7 +315,11 @@ Note that the diagnostics bundle is intended for debugging purposes only, its st
 IMPORTANT: The configuration files in the archive may have credentials in *plain text*.
 These will need to be manually redacted before the archive is shared.
 
-The diagnostics command is also able to collect `pprof` data in the archive if the `--pprof` flag is passed:
+The diagnostics command is also able to collect `pprof` data in the archive if the `--pprof` flag is added when collecting the diagnostic.
+
+To collect the `pprof` data:
+- Enable/add `agent.monitoring.pprof.enabled: true` to `elastic-agent.yml`
+- Restart the Elastic Agent
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc:322]:
```suggestion
- Restart the {agent}
```

**Review by dedemorton — COMMENTED** [review]
Minor change. We're going to be linting for product names soon, so should use attributes.

**Comment by dedemorton** [comment]
@belimawr I committed the small change I suggested. Is this ready to approve/merge?

**Pull request merged**